### PR TITLE
feat: [CGC-19] Add pipeline to the project

### PR DESCRIPTION
**Pull request by edvinstava**

### CODE REVIEW

File: .github/workflows/verify-app.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: 'Capture Growth Chart: verify app'
+
+on:
+  push:
+    branches:
+
+jobs:
+  install:
```
Comment (henrikmv) [inline .github/workflows/verify-app.yml:8]: More specific names for the jobs:

- `install` -->   `install_dependencies`
- `lint` -->   `lint_code`
- `typescript` -->   `typescript_check`

File: .github/workflows/verify-app.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+  lint:
```
Comment (henrikmv) [inline .github/workflows/verify-app.yml:26]: Add the name key for readability. Add it for all of the three jobs:

- `name: Install Dependencies`
- `name: Lint Code`
- `name: TypeScript Check`

File: .github/workflows/verify-app.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
```
Comment (henrikmv) [inline .github/workflows/verify-app.yml:15]: Using `node-version: ${{ matrix.node-version }}` allows us to easily modify or add support for different Node.js versions without the need to directly edit the configuration file. Please change the node version config in the lint job also.

Comment (henrikmv) [reply .github/workflows/verify-app.yml:15]: You can add a **strategy** at the bottom of the page: 
```
strategy:
    matrix:
      node-version: [18.x]
```

**Review by henrikmv — APPROVED** [review]
Just a few comments to improve readability

File: package.json [hunk 4]
```diff
@@ -20,26 +20,27 @@
...
     "@dhis2/d2-i18n": "^1.1.3",
+    "@dhis2/prop-types": "^3.1.2",
     "@dhis2/ui": "^9.0.1",
     "@material-ui/core": "3.9.4",
     "@tanstack/react-query": "^4",
     "@types/chart.js": "^2.9.41",
     "chart.js": "^3.0.0",
     "classnames": "^2.5.1",
+    "prop-types": "15.7.2",
+    "react": "^16.8.0",
```
[the remark this answers is not in the document]
Comment (edvinstava) [reply package.json:37]: Had to downgrade to remove warning from yarn install in pipeline

**Pull request merged**